content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md

```diff
@@ -117,43 +117,41 @@ If **Load Balancer** is selected, the following must be true:<p></p><br />
 
 # Deploying a Palette Virtual Cluster
 
-1. From the slide menu, select the **Clusters** tab and click the **Palette Virtual Clusters** tab to list the available Palette Virtual Clusters, and then select **Add New Palette Virtual Cluster**.
+1. From the **Main** menu, click **Clusters** and select a project from the drop-down menu. 
 
```

Comment:
Let's invert the actions - select a project then click Clusters from the menu.

The sequence doesn't matter if the user is already in the context of a project, but it matters if the user is a Tenant Admin. If a Tenant Admin clicks Clusters, then selected a project they see the Project Overview page, and have to click Clusters once again.

Reply:
Great suggestion Amarjeet 😄

Reply:
That makes sense - I'll switch the actions.